ArduCopter/AP_Arming.cpp

```diff
@@ -633,7 +633,13 @@ bool AP_Arming_Copter::arm_checks(AP_Arming::Method method)
         SRV_Channels::set_emergency_stop(false);
         // if we are using motor Estop switch, it must not be in Estop position
     } else if (rc().find_channel_for_option(RC_Channel::AUX_FUNC::MOTOR_ESTOP) && SRV_Channels::get_emergency_stop()){
-        gcs().send_text(MAV_SEVERITY_CRITICAL,"Arm: Motor Emergency Stopped");
```

Comment:
Good, unrelated, change.  Should be in a separate PR to simplify this one.

Reply:
fixed it!

Reply:
Not seeing that PR.

Reply:
I misread.
Done!